docs/source/operators/deploy-kubernetes.md

```diff
@@ -15,49 +16,414 @@ The following sample kernel specifications apply to Kubernetes deployments:
 - spark_python_kubernetes
 - spark_scala_kubernetes
 
-As with all Kubernetes deployments, Enterprise Gateway is built into a docker image. The base Enterprise Gateway image is [elyra/enterprise-gateway](https://hub.docker.com/r/elyra/enterprise-gateway/) and can be found in the Enterprise Gateway dockerhub organization [elyra](https://hub.docker.com/r/elyra/), along with other kubernetes-based images. See [Docker Images](../contributors/docker.md) for image details.
+Enterprise Gateway deployment is using  [elyra/enterprise-gateway](https://hub.docker.com/r/elyra/enterprise-gateway/) image under Enterprise Gateway dockerhub organization [elyra](https://hub.docker.com/r/elyra/) along with other kubernetes-based images. See [Docker Images](../contributors/docker.md) for image details.
 
 When deployed within a [spark-on-kubernetes](https://spark.apache.org/docs/latest/running-on-kubernetes.html) cluster, Enterprise Gateway can easily support cluster-managed kernels distributed across the cluster. Enterprise Gateway will also provide standalone (i.e., _vanilla_) kernel invocation (where spark contexts are not automatically created) which also benefits from their distribution across the cluster.
 
-## Deploying Enterprise Gateway on Kubernetes
+We are using helm templates to manage Kubernetes resource configurations which, allows enduser easily customize their Enterprise Gateway deployment.
```

Comment:
couple of minor grammatical changes requested:

1. 'configurations which, allows' -> 'configurations, which allows'
2. 'allows enduser easily customize' -> 'allows end user to easily customize'

Reply:
Thanks @bloomsa.

As a follow-on...

> 'allows enduser easily customize' -> 'allows end user to easily customize'

-> 'allows an end user to easily customize'